test/migration_generator_test.exs

```diff
@@ -2345,4 +2333,11 @@ defmodule AshPostgres.MigrationGeneratorTest do
                ~S[modify :post_id, references(:posts, column: :id, name: "comments_post_id_fkey", type: :uuid, prefix: "public")]
     end
   end
+
+  defp get_migration_files(opts) do
```

Comment:
@zachdaniel What do you think of this helper function and default opts in setup? I like this because it adds clarity in test. I could refactor the whole file if welcomed. 🐛

Reply:
I like it 👍 lets refactor that in a different PR though.